docs/features/software-templates/writing-templates.md

```diff
@@ -46,6 +46,18 @@ spec:
           ui:options:
             catalogFilter:
               kind: Group
+        user:
+          title: User
+          type: string
+          description: Owner of the component
+          ui:field: EntityPicker
+          ui:options:
+            allowArbitraryValues: false
+            nameTemplate: ${entity.spec.profile.displayName} (${entity.spec.profile.email})
```

Comment:
Hm. `${` vs `${{` will probably be a source of support requests i bet. 🤔

Reply:
If we choose to use regexp, we can also use `${{` as marker.